src/output-json-alert.c

```diff
@@ -147,12 +147,10 @@ static void AlertJsonSsh(const Flow *f, JsonBuilder *js)
     void *ssh_state = FlowGetAppState(f);
     if (ssh_state) {
         void *tx_ptr = rs_ssh_state_get_tx(ssh_state, 0);
-        json_t *tjs = rs_ssh_log_json(tx_ptr);
-        if (unlikely(tjs == NULL))
-            return;
-
-        jb_set_jsont(js, "ssh", tjs);
-        json_decref(tjs);
+        jb_open_object(js, "ssh");
+        // log even if no data
+        rs_ssh_log_json(tx_ptr, js);
```

Comment:
Return value is not tested
We may have `"ssh" : {}` in the json

Reply:
Look at the mark support https://github.com/OISF/suricata/commit/b3b5802c85c8e893fdf6ae144880836bff0a16ee#diff-e2a291d82d2cff36c51007fafaa79b86R515

Reply:
Thanks